lab-yunjoo/entry.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
```

Comment:
Entry files hold backend code that tells webpack what to bring in. They should just be a list of require statements. 

Everything that you do below w/ angular should be done in a separate file, then required in.